bin/composer-auth.sh

```diff
@@ -20,8 +20,9 @@ then
 	cat <<- EOF >> auth.json
 		{
 			"http-basic": {
-			  "composer.wp.dsd.io": {"username": "$COMPOSER_USER","password": "$COMPOSER_PASS"},
-        "composer.deliciousbrains.com": {"username": "$AS3CF_PRO_USER","password": "$AS3CF_PRO_PASS"}
+				"composer.wp.dsd.io": {"username": "$COMPOSER_USER","password": "$COMPOSER_PASS"},
+				"composer.deliciousbrains.com": {"username": "$AS3CF_PRO_USER","password": "$AS3CF_PRO_PASS"},
+				"connect.advancedcustomfields.com": {"username": "$ACF_PRO_LICENSE", "password": "https://intranet.justice.gov.uk"}
```

Comment:
Hi @EarthlingDavey - Did you set up basic authentication using our domain as a password?

Reply:
Our security department will flag this as the 'password' has a value. We might want to stop them from messaging us by popping the value in a hidden variable.

Reply:
Hey, good spot. I didn't set the password, it's how acf does the auth.

https://www.advancedcustomfields.com/resources/installing-acf-pro-with-composer/#build-systems-and-environment-variables

Do you think commenting would be sufficient, or set an environment variable?